mercapi/mercapi.py

```diff
@@ -22,19 +24,28 @@ class Mercapi:
 
     **Avoid instantiating this class more than once in a single runtime.**
     """
-
-    _user_agent = (
-        "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"
-    )
-    _headers = {
-        "User-Agent": _user_agent,
-        "X-Platform": "web",
-    }
-
-    def __init__(self):
+    
+    def __init__(
+        self,
+        *,
+        proxies: typing.Optional[ProxiesTypes] = None,
+        user_agent: str = (
```

Comment:
Also, please follow the convention of optional parameters, and set `None` as a default value here. Then, the constructor should set the default UA if `user_agent` parameter is `None`.

```python
if not user_agent:
    self._headers = { "User-Agent": "Mozilla/5.0 ..." }
```

Reply:
Thanks for pointing that out. I have corrected it that way, is that ok?